Tests/DatadogTests/DatadogObjc/DDGlobalTests.swift

```diff
@@ -43,25 +44,12 @@ class DDGlobalTests: XCTestCase {
 
     func testWhenRUMMonitorIsNotSet_itReturnsNoOpImplementation() {
         XCTAssertTrue(DatadogObjc.DDGlobal.rum.swiftRUMMonitor is DDNoopRUMMonitor)
-        XCTAssertTrue(Global.rum is DDNoopRUMMonitor)
     }
 
     func testWhenRUMMonitorIsSet_itSetsSwiftImplementation() throws {
```

Comment:
test name is not right now. We're testing if initialisation is setting correct swifRUMMonitor if I read this correctly.